### PR TITLE
Style SignIn and Onboarding after moving under auth path

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/123?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #113** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/113?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #114** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/114?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #115** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/115?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #116** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/116?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #118** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/118?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #121** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/121?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
              * **PR #122** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/122?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #123** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/123?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
            * **PR #119** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/119?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/123?utm_source=stack-comment).

File: packages/app/features/auth/onboarding/onboarding-form.tsx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        key_type: 'ES256',
+      },
+      key_slot: keySlot,
+    })
+
+    if (error) {
+      throw error
+    }
+
+    if (__DEV__) {
```
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/onboarding-form.tsx:77]: we can drop this now, move it to the secret shop.

File: packages/app/features/auth/onboarding/onboarding-form.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        client: testClient,
+        address: senderAddress,
+        tokenAddress: usdcAddress[baseMainnetClient.chain.id],
+        value: BigInt(100e6),
+      })
+    }
+
+    await sendAcctsRefetch()
+
+    // await createAccountUsingEOA(_publicKey)
```
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/onboarding-form.tsx:93]: can also drop this and the function.

File: packages/app/features/auth/onboarding/onboarding-form.tsx
Comment (0xBigBoss) [inline packages/app/features/auth/onboarding/onboarding-form.tsx:1]: thanks for cleaning this up.

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 28, 12:46 AM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/123).

**Pull request merged**